pkg/rancher-desktop/components/Snapshots.vue

```diff
@@ -78,6 +80,11 @@ export default Vue.extend<Data, Methods, Computed, never>({
         this.$store.dispatch('snapshots/fetch');
       }, 1500);
     },
+    showWhen() {
```

Comment:
I think this would be better named `now()` or something along those lines, because it just always returns the current time.

Reply:
sure